design/explainer.md

```diff
@@ -0,0 +1,128 @@
+# Noosphere Explainer
+
+*Last updated: July 2nd, 2022*
+
+Noosphere is a massively-multiplayer knowledge graph. The technical pillars that Noosphere builds upon are:
+
+* [Public key infrastructure](https://en.wikipedia.org/wiki/Public_key_infrastructure)
+* [Content addressing](https://en.wikipedia.org/wiki/Content-addressable_storage)
+* [Immutable data](https://en.wikipedia.org/wiki/Immutable_object)
+* [P2P routing and discovery](https://en.wikipedia.org/wiki/Peer-to-peer)
+
+Above this substructure, Noosphere gives users:
+
+* Entry to a zero-trust, **decentralized network** of self-sovereign nodes
+* **Human-readable names** for peers and their public content
+* **Local-first authoring** and offline-available content with conflict-free synchronization
+* A complete, space-efficient **revision history** for any content
+* Coherence and **compatibility with the hypertext web**
+
+You can think of it like a world-wide Wiki.
+
+## How it feels…
+
+Subconscious user Bob authors a note about cats, formatted as Subtext:
+
+> *Bob's notebook · /cat-thoughts*
+>
+> I love cats. I love every kind of cat.
```

Comment:
Tiny nitpick, but these might render better as tables in Markdown.

Otherwise, nicely done on replicating the formatting within the limitations of Markdown! 👏👏👏

Reply:
Tried out tables -- looks good, and using `<br />` for new lines in the "note" based illustrations makes it distinct from the "data" entries/row ("links", address book/names), which is a good fit

Reply:
Thanks for the feedback!